docs/man1/rst_deck.1

```diff
@@ -86,95 +86,3 @@ rst_deck \fI\,/path/to/history/HISTORY.DATA\/\fP rst/HISTORY:30 \fI\,/path/to/rs
 1: The program has a compiled list of keywords which will be retained in the
 .IP
 SOLUTION section. The current value of that list is: RPTRST
-.PP
-\&../../opm\-parallel/bin/rst_deck: invalid option \fB\-\-\fR 'v'
-\&../../opm\-parallel/bin/rst_deck: invalid option \fB\-\-\fR 'e'
-\&../../opm\-parallel/bin/rst_deck: invalid option \fB\-\-\fR 'r'
-\&../../opm\-parallel/bin/rst_deck: invalid option \fB\-\-\fR 'i'
-\&../../opm\-parallel/bin/rst_deck: invalid option \fB\-\-\fR 'o'
-\&../../opm\-parallel/bin/rst_deck: invalid option \fB\-\-\fR 'n'
-.PP
-The rst_deck program will load a simulation deck and parameters for a restart
-and reformat the deck to become a restart deck. Before the updated deck is
-output the program will update the SOLUTION and SCHEDULE sections. All keywords
-from the SOLUTION section will be cleared out(1) and a RESTART keyword will be
-inserted. In the SCHEDULE section the program can either remove all keywords up
-until the restart date, or alternatively insert SKIPREST immediately following
-the SCHEDULE keyword.
-.PP
-When creating the updated restart deck the program can either link to unmodified
-include files with INCLUDE statements, create a copy of deck structure in an
-alternative location or create one large file with all keywords in the same
-file. Apart from the alterations to support restart the output deck will be
-equivalent to the input deck, but formatting is not retained and comments have
-been stripped away.
```

Comment:
You added this section in #2997.  Why is it appropriate to remove it now?

Reply:
Because this section and the following somehow where created by help2man due to me passing wrong options and I failed to notice that back then. Fortunately, I checked this manually this time. The first few lines are complaining about me passing `--version`.  The rest is already [here](https://github.com/OPM/opm-common/blob/49f06f0cb7f643d0692fa6afa610e4b10dd19ce1/docs/man1/rst_deck.1#L5-L18).